### DISCUSSION TITLE
Low Battery level → notification

### DISCUSSION THREAD

**Discussion by netikras** [NightscoutFoundation/xDrip#1953] [Ideas]
I know users can see the transmitter's battery charge (%) when they open an app. However, some (many? most?) users spend their day w/o opening the app Activity and only rely on notifications (they are super handy!). Hence it's very easy to miss when the transmitter's battery gets low. And it's super annoying when a transmitter dies quietly when you're far away from any USB outlet (or don't have the charger on you)

My proposal is to create a warning notification (with a respective icon drawing appropriate attention) that sticks around in the notification bar/panel when/for as long as the transmitter reports its battery charge below the preconfigured level (e.g. 15%). The notification should be updated periodically with the current transmitter's charge.

**Comment by Navid200** [comment]
What datasource do you use?
If Dexcom G5 or G6, this already exists:

Settings -> G5/G6 Debug Settings -> Adjust battery warning level

**Reply by netikras** [reply]
MiaoMiao + libre

**Reply by netikras** [reply]
![received_473292394252929](https://user-images.githubusercontent.com/2045031/154811731-d1a2e50f-561c-4a7b-a240-f56700073b89.jpeg)
if I'm seing it right, it's enabled, but there are no transmitter battery related notifications popping up.

**Comment by Navid200** [comment]
@psonnera @keencave @tzachi-dar Sorry for tagging.
I know the optional warning exists already for Dexcom.  Does it not exist for the Libre data source option?

**Reply by psonnera** [reply]
Tried on my follower and it doesn't trigger anything. The issue mainly impacts MM2 users as battery lifetime is less than sensor lifetime.

**Reply by psonnera** [reply]
As I was playing with Wear smartwatches I had the alarm trigger several times when the watch itself went below the alarm threshold. At this point I believe the bridge definition is a historical definition.
